docs_versioned_docs/version-ros2humble/ros/config/live.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: View and Edit the YAML
+sidebar_label: Live YAML
+sidebar_position: 4
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+Users familiar with Clearpath robots, know we provide desktop packages that facilitate the viewing of robot models in RViz without needing to have a real robot. In ROS 2, the `view_model.launch.py` node will check and update the model and display the new changes in RViz.
```

Comment:
Similar comment to another file in this PR, suggesting a change to "Clearpath's robots"